jolly-jellyfish/src/static/themes/console.css

```diff
@@ -0,0 +1,55 @@
+/**
+ * Green on black monospace theme.
+ * Every paragraph begins with "> " and every header with ">>> ".
+ * The final paragraph ends with a blinking cursor, as well as every header.
+ * Headers are constantly deleted and rewritten, and must be written as:
+ *     <h1><span><span>Header Text</span</span></h1>
```

Comment:
Are you sure we can't remove one of these spans? If we're using them as unique ids then we could drop a class on the outer one and use `h1 > span.class` (I think... CSS is not my forte)

Reply:
Only one is needed here, but two are required for the hotdog theme.

Reply:
For simplicity, I will only select the first one here though, that is a good idea

Reply:
Done